doc/docs.md

```diff
@@ -326,6 +326,16 @@ fn main() {
 }
 ```
 
+The values of multiple variables can be changed in one line. In this way, their values can be swapped without an intermediary variable.
```

Comment:
This line is too long and should be wrapped. Perhaps after the first sentence?

Reply:
Thank you! It seems to pass now. Are there any other problems?

Reply:
no, everything seems fine